src/libraries/Common/src/Interop/Browser/Interop.Runtime.cs

```diff
@@ -330,11 +330,9 @@ internal static string GetCallSignature(IntPtr method_handle)
 
             var mb = MethodBase.GetMethodFromHandle(tmp.handle);
 
-            string res = "";
-            var parms = mb?.GetParameters();
-            if (parms == null)
-                return res;
-            foreach (var p in parms)
+            char[] res = new char[20];
```

Comment:
Get the correct length from `parms.Length` and return early on 0

Reply:
This should be taken care of in the code base now.